tests/core/mem-cgroup-disabled/task.yaml

```diff
@@ -112,7 +106,11 @@ execute: |
       snap remove test-snapd-simple-service
       
       # revert back to normal pc gadget without the command line
-      snap revert pc
+      snap revert --no-wait pc
```

Comment:
I thoughtt that with the changes mvo landed  in cmd we wouldn't need no-wait anymore?

Reply:
Some of the spread test changes are no longer neccessary after the improvements made, and I am currently retesting how many of the changes still makes sense